SingularityService/src/main/java/com/hubspot/singularity/data/SingularityValidator.java

```diff
@@ -114,6 +114,10 @@ public SingularityRequest checkSingularityRequest(SingularityRequest request, Op
     checkBadRequest(request.getId() != null && !StringUtils.containsAny(request.getId(), JOINER.join(REQUEST_ID_ILLEGAL_CHARACTERS)), "Id can not be null or contain any of the following characters: %s", REQUEST_ID_ILLEGAL_CHARACTERS);
     checkBadRequest(request.getRequestType() != null, "RequestType cannot be null or missing");
 
+    if (request.getOwners().isPresent()) {
+      checkBadRequest(!request.getOwners().get().contains(null), "Request owner(s) must not be null");
```

Comment:
wording here is a little vague (i.e. `owners: null` is actually valid, but the message makes it sound like it wouldn't be) -- perhaps something like "Request owners cannot contain null values" ?